OpenRA.Mods.Common/Traits/GivesBounty.cs

```diff
@@ -89,7 +89,7 @@ int GetDisplayedBountyValue(Actor self)
 
 		void INotifyKilled.Killed(Actor self, AttackInfo e)
 		{
-			if (e.Attacker == null || e.Attacker.Disposed)
+			if (e.Attacker == null || e.Attacker.Disposed || !self.World.WorldActor.Trait<GlobalBounty>().Bounty)
```

Comment:
Can you please change this to
```csharp
var globalBounty = self.World.WorldActor.TraitOrDefault<GlobalBounty>();
if (e.Attacker == null || e.Attacker.Disposed || (globalBounty != null && !globalBounty.Bounty))
```

This then defaults the bounty to enabled for mods that do not want to provide a global option to disable it.